app/models/patient.rb

```diff
@@ -0,0 +1,18 @@
+class Patient < ApplicationRecord
+  GENDERS = %w[male female transgender].freeze
+  STATUSES = %w[active dead migrated unresponsive inactive]
+
+  belongs_to :address, optional: true
+  has_and_belongs_to_many :phone_numbers
+
+  validates_presence_of :created_at, :updated_at
```

Comment:
This will cause us to manually set these fields before we can save any new records, not ideal.

Reply:
These fields need to be set on the front-end devices, and are required params in the API. But we could perhaps look into naming these fields differently in the API and model.